spec/statsd/sender_spec.rb

```diff
@@ -46,17 +47,51 @@
         end.to raise_error(ArgumentError, /Sender already started/)
       end
     end
+
+    context 'when flush_interval is set' do
+      let(:flush_interval) { 0.001 }
+
+      it 'starts a worker thread and a flush timer thread' do
+        mutex = Mutex.new
+        cv = ConditionVariable.new
+        expect(subject).to receive(:flush).and_wrap_original do
+          mutex.synchronize { cv.broadcast }
+        end
+
+        expect do
+          subject.start
+        end.to change { Thread.list.size }.by(2)
+
+        # wait a second or until #flush is called
+        mutex.synchronize { cv.wait(mutex, 1) }
```

Comment:
This is really minor, but maybe it's better to have the mutex/CV protect a boolean value.  Then if flush is called before the wait, the test will still succeed (because the boolean value will be set already).

Reply:
Thanks to your comment, I noticed that this test didn't work expectedly. Even if `flush` wasn't called, the test still succeeded because `stop` called `flush`.
I fixed it in the commit https://github.com/DataDog/dogstatsd-ruby/pull/231/commits/657623aea63e5edcd523ca532c44af3e9a9c7abf.